index.js

```diff
@@ -12,9 +84,149 @@ function toggleMenu() {
   document.querySelector('.top-navigation .name .fa-times').classList.toggle('menu');
   document.querySelector('.top-navigation').classList.toggle('fullHeight');
 }
-// Hide menu on Click
-menusArray.forEach((element) => {
+
+menusArray.forEach((element) => { // Hide menu on Click
   element.addEventListener('click', () => {
     toggleMenu();
   });
-});
\ No newline at end of file
+});
+
+let modalContent = { // launch projects modal
+  header: '',
+  technologies: [],
+  description: '',
+  img: '',
+};
+
+document.querySelector('.modal-content i').addEventListener('click', (event) => {
+  event.preventDefault();
+  modalContent = {
+    header: '',
+    technologies: [],
+    description: '',
+    img: '',
+
+  };
+  document.querySelector('.modal').classList.toggle('hide');
+});
+
+function launchProject(id) {
+  if (id >= 0) {
+    modalContent.header = projects[id].name;
+    modalContent.technologies = projects[id].technologies;
+    modalContent.description = projects[id].description;
+    modalContent.img = projects[id].image;
+
+    document.querySelector('.modal').classList.toggle('hide');
+    document.querySelector('.modal-content h2').innerText = modalContent.header;
+    document.querySelector('.modal-content-body > div:first-child').style.setProperty('background-image', `url("${modalContent.img}")`);
+    let ulContent = '';
+    modalContent.technologies.forEach((item) => {
+      ulContent += `<li><span>${item}</span></li>`;
+    });
+    const ul = document.querySelector('.modal-content ul');
+    ul.innerHTML = ulContent;
+    document.querySelector('.modal-content-body p').innerText = modalContent.description;
+  }
+}
```

Comment:
- [ ] Consider making your modal work properly because it is working well only on the main card but when I click on the first card at the bottom, it appears without content, the following image illustrates what I mean.

> ![image](https://user-images.githubusercontent.com/53402600/196640218-629d719b-056c-4e47-ba37-03dcfb02bfc5.png)